### ISSUE TITLE
Found Inf or NaN in globalnorm during training the model

### ISSUE THREAD

**Issue by SiddeshkanthLogonathan** [Danyleb/Lstm-Variational-Auto-encoder#10]
during training there happens to be an error and I can't train  the ltsm-vae model. I've read some recommendations such as reduce batch size or reduce learning rate or use AdamOptimizer. Even after all of the possible solutions, I still get this error.

![Capture](https://user-images.githubusercontent.com/49587560/117628178-27b0f600-b179-11eb-89ba-69253313f99c.PNG)
